### ISSUE TITLE
Support custom index.html entrypoint

### ISSUE THREAD

**Issue by danneu** [elm-lang/elm-reactor#160]
As an Elm beginner, I quickly got to a point where I had a custom `index.html` file (to link in my custom CSS files) which mounted my `src/Main.elm` file. For example, I almost always start a project with Twitter Bootstrap so I have some basic UI components to play with while prototyping as it's a chore to prototype with unstyled HTML.

My envisioned workflow would be to run `elm-reactor`, open my `index.html`, and refresh the page as I make changes to my `src/**.elm` files (bonus: hot reload).

elm-reactor doesn't support this afaict, and I'm not fully understood in the intended scope of elm-reactor, but it seems like this an obvious responsibility for elm-reactor.

So far I have a script that watches the filesystem and re-runs `elm-make` on change. And it doesn't seem like this is uncommon or already supported by elm-reactor (https://langrisa.eu/auto-recompile-using-your-own-html/).

What do you think?

**Comment by jvoigtlaender** [comment]
Have you seen https://github.com/elm-lang/elm-reactor#debugging-code-embedded-in-html?

**Comment by tmcw** [comment]
That bit got removed from the README. I've dug through the code, and can't find custom index.html support. Afaik, custom HTML is required to use [ports](https://guide.elm-lang.org/interop/javascript.html).

**Comment by jvoigtlaender** [comment]
Yes, it appears that got removed at some point in the last 10 months. The reactor is undergoing a revamp. When that is done, you can probably expect complete instructions for this use case again. In the meantime, you could ask on the mailing list how people cope with this issue right now.

**Comment by shishkin** [comment]
For OSX users, [fswatch](https://github.com/emcrisostomo/fswatch) seems to do the trick of kicking `elm-make` to recompile the sources for `elm-reactor` to serve:

```
fswatch -o ./src/ | while read; do elm-make src/Main.elm --output out/main.js; done
```

Although any static web server would do instead of `elm-reactor` in this case.

**Comment by halcwb** [comment]
Is there any progress on this issue? There are workarounds, however, they very much depend on all sorts of external factors, like OS, node version, how node is installed (nvm?) etc... So, I more reliable solution would be great.
